server.go

```diff
@@ -907,9 +914,18 @@ func SaveMultipartFile(fh *multipart.FileHeader, path string) error {
 	if err != nil {
 		return err
 	}
-	defer ff.Close()
+
 	_, err = copyZeroAlloc(ff, f)
-	return err
+	if err != nil {
+		return err
+	}
+
+	err = ff.Close()
+	if err != nil {
+		return err
+	}
```

Comment:
Now `ff.Close()` isn't called if `copyZeroAlloc` fails which could cause files to stay open (multipart files are backed by os files). I'm not sure why this wasn't an issue before. Doesn't changing it to `defer ff.Close() //nolint:errcheck` fix the issue?

Reply:
Done